cmake/emu/armfvp.cmake

```diff
@@ -10,11 +10,24 @@ find_program(
   NAMES ${ARMFVP_BIN_NAME}
   )
 
+if(CONFIG_ARMV8_A_NS)
+  set(armfvp_bl1_path $ENV{ARMFVP_BL1_PATH})
+  set(armfvp_fip_path $ENV{ARMFVP_FIP_PATH})
```

Comment:
Seems users already need to enable ARMV8_A_NS using Kconfig, has it been considered to offer the possibility of specifying the ARMFVP_BL1_PATH and ARMFVP_FIP_PATH using Kconfig ?

Reply:
These files are firmwares to be loaded in the emulator for runtime, for me Kconfig needs to be related to Zephyr, here the link is indirect.

Reply:
fair enough.

Then maybe add this setting to the CMake cache.
The problem with an environment variable is that it always needs to be set, and be set to the same value when CMake re-run.
Example:
```
# Open terminal.
$ export ARMFVP_BL1_FILE=/<path>/file.bin
$ cmake -GNinja -Bbuild ...
$ ninja -Cbuild
```
as long as user edit source files and build using ninja, everything is fine, but if the user edits a cmake file (like adding an extra file to a library) and build in another terminal, like:
```
# Edit a cmake file
# Open second terminal.
$ ninja -Cbuild
# --> CMake re-run --> boom, because ARMFVP_BL1_FILE is not defined in this environment
```
similar may happen to users switching between IDE and command line, or in general have multiple setups.

I would suggest as minimum to add ARMFVP_BL1_FILE and ARMFVP_FIP_FILE to the CMake cache.
You can still allow initial value to be taken from the environment.

This ensures that CMake re-runs doesn't suddenly break, and a user may still update the CMakeCache with an updated file path if they so wish.

Reply:
Sure, will add the change